src/ui/components/Playlist.tsx

```diff
@@ -142,13 +143,15 @@ export const Playlist: FunctionComponent<Props> = ({
           <BicolorArrowRight />
         </ScrollButtonForNotTouchDevice>
       ) : null}
-      <FlatList
+      <FlashList
         onLayout={({ nativeEvent }) => {
           setPlaylistWidth(nativeEvent.layout.width)
         }}
         testID={testID}
         ref={flatListRef}
         scrollEnabled={isTouch}
+        drawDistance={Dimensions.get('window').width / 2 / 3}
```

Comment:
might be a bit agressive but ensures we render only 1.5 screens of items basically
scrolling seems good otherwise